.gitlab-ci.yml

```diff
@@ -667,8 +668,10 @@ library:ci-cross-crypto:
 library:ci-fcsl-pcm:
   extends: .ci-template
 
+# We cannot use flambda due to a bug, see
+# and https://github.com/coq/coq/pull/11916#issuecomment-609977375
```

Comment:
Should there be something between "see" and "and"?

Reply:
Sorry, the `and` is spurious from the previous version of this PR where we tried to increase stack size; amended.